spec/support/active_job.rb

```diff
@@ -4,7 +4,7 @@
   config.include ActiveJob::TestHelper, type: :feature
 
   config.before(:each) do |example|
-    if [:job, :feature].include? example.metadata[:type] || example.metadata[:active_job]
+    if (ActiveJob::Base.queue_adapter.respond_to?(:perform_enqueued_jobs)) && ([:job, :feature].include?(example.metadata[:type]) || example.metadata[:active_job])
```

Comment:
I don't know what this is really about, and would like to know.

Reply:
for the tests where we flop over to `queue_adapter = :delayed_job` to test the delayed job activerecord stuff, `clear_enqueued_jobs` and `clear_performed_jobs` don't exist

I guess it could be alternately be written as
```
if whatever
  clear_enqueued_jobs if respond_to?(:clear_enqueued_jobs)
end
```

(does that work? just typing off the cuff here)

but that seemed a little weirder because if `clear_enqueued_jobs` somehow stopped existing because it got renamed that vestigal line would be there forever